jupyter_book_to_htmlbook/file_processing.py

```diff
@@ -241,6 +241,8 @@ def process_chapter(toc_element,
     chapter = process_footnotes(chapter)
     chapter = process_admonitions(chapter)
     chapter = process_math(chapter)
+    # note: best to run examples before code processing
+    chapter = process_code_examples(chapter)
     chapter = process_code(chapter, skip_cell_numbering)
```

Comment:
Question: as-is, this also means that examples will get cell numbering (e.g., `In [1]: def x()`) included in examples. Do we want to _not_ do this? Or leave this to be dealt with in production?